examples/runners/wgpu/src/lib.rs

```diff
@@ -57,6 +57,46 @@ pub enum RustGPUShader {
 }
 
 fn shader_module(shader: RustGPUShader) -> wgpu::ShaderModuleDescriptor<'static> {
+    #[cfg(not(any(target_os = "android", target_arch = "wasm32")))]
+    {
+        use spirv_builder::SpirvBuilder;
+        use std::borrow::Cow;
+        use std::path::{Path, PathBuf};
+        let crate_name = match shader {
+            RustGPUShader::Simplest => "sky-shader",
+            RustGPUShader::Sky => "simplest-shader",
+            RustGPUShader::Compute => "compute-shader",
+            RustGPUShader::Mouse => "mouse-shader",
+        };
+        let manifest_dir = env!("CARGO_MANIFEST_DIR");
+        let crate_path = [
+            Path::new(manifest_dir),
+            Path::new(".."),
+            Path::new(".."),
+            Path::new("shaders"),
+            Path::new(crate_name),
+        ]
+        .iter()
+        .copied()
+        .collect::<PathBuf>();
```

Comment:
Isn't this shorter as a chain of `.join(...)` calls? Tho I suppose that may be less efficient hmpf.

Reply:
yeah, probably, idk, I tend to over-optimize things that really I probably should just write the clean readable way :|